secureli/services/language_support.py

```diff
@@ -212,10 +219,10 @@ def _build_pre_commit_config(self, languages: list[str]) -> BuildConfigResult:
                     )
                 ) if result.linter_config.successful else None
                 data = yaml.safe_load(result.config_data)
-                for config in data["repos"]:
-                    config_data.append(config)
+                if data["repos"] != None:
+                    for config in data["repos"]:
+                        config_data.append(config)
```

Comment:
I think I would suggest either this
```suggestion
                if data["repos"]:
                    config_data.extend(data["repos"])
```
or this
```suggestion
                config_data += data["repos"] or []
```